src/settings-ui/Settings.UI.Library/ViewModels/MeasureToolViewModel.cs

```diff
@@ -162,9 +164,21 @@ public HotkeySettings ActivationShortcut
         public void NotifyPropertyChanged([CallerMemberName] string propertyName = null)
         {
             OnPropertyChanged(propertyName);
+
+            if (propertyName == nameof(ShowContinuousCaptureWarning))
+            {
+                // Don't trigger a settings update if the changed property is for visual notification.
+                return;
+            }
+
             SettingsUtils.SaveSettings(Settings.ToJsonString(), MeasureToolSettings.ModuleName);
         }
 
+        public bool ShowContinuousCaptureWarning
+        {
+            get => IsEnabled & ContinuousCapture;
```

Comment:
that's a bitwise and, no?
```suggestion
            get => IsEnabled && ContinuousCapture;
```

Reply:
Yep.

Reply:
@yuyoyuppe
Thank you for commiting.